src/System.Net.WebSockets.WebSocketProtocol/tests/System.Net.WebSockets.WebSocketProtocol.Tests.csproj

```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>$(NetMinimum);net48</TargetFrameworks>
```

Comment:
It looks like our other tests target `$(NetFrameworkMinimum)`.  Any reason we didn't do that here?

Reply:
@ericstj there are some APIs that are unavailable in net462. In the 5.0 code base we are also targeting net48 in the tests:  https://github.com/dotnet/runtime/blob/4aadfea70082ae23e6c54a449268341e9429434e/src/libraries/System.Net.WebSockets.WebSocketProtocol/tests/System.Net.WebSockets.WebSocketProtocol.Tests.csproj#L3

Specifically, these APIs are unavailable:
- `LocalEndPoint`: https://github.com/dotnet/maintenance-packages/blob/af57e1f49e3aba0fc03cb9a3232d25f2ed329890/src/System.Net.WebSockets.WebSocketProtocol/tests/WebSocketCreateTest.cs#L125
- `AcceptAsync`: https://github.com/dotnet/maintenance-packages/blob/af57e1f49e3aba0fc03cb9a3232d25f2ed329890/src/System.Net.WebSockets.WebSocketProtocol/tests/WebSocketCreateTest.cs#L126
- `SendAsync`: https://github.com/dotnet/maintenance-packages/blob/af57e1f49e3aba0fc03cb9a3232d25f2ed329890/src/System.Net.WebSockets.WebSocketProtocol/tests/WebSocketCreateTest.cs#L130
- All the 3 above are also used in this other unit test: https://github.com/dotnet/maintenance-packages/blob/af57e1f49e3aba0fc03cb9a3232d25f2ed329890/src/System.Net.WebSockets.WebSocketProtocol/tests/WebSocketCreateTest.cs#L183

Reply:
Consider adding a comment when you have a difference like this.  It might help others confused by the difference.